testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -124,6 +124,26 @@ impl EmitJobRequest {
             },
         }
     }
+
+    pub fn fixed_tps(instances: Vec<Instance>, tps: u64) -> Self {
+        let mut worker_num: usize = 10;
+        while instances.len() * worker_num > tps as usize {
+            if worker_num == 1 {
+                break;
+            }
+            worker_num -= 1;
+        }
+        let wait_time = (instances.len() * worker_num * 1000_usize / tps as usize) as u64;
```

Comment:
Can you run this with `--run perf -- --tps 5` and provide link to dashboard and output?